docs/using/cli.md

```diff
@@ -2,28 +2,32 @@
 
 # Command-Line
```

Comment:
They are most definitely (2), since thats literally what the `jcache-cli` directives do, execute the commands linerally and show the input/output.
This file is basically a "pseudo" MyST notebook, where the `{jcache-cli}` are special `{code-cell}`, with a specific input/output format